.bcr/metadata.template.json

```
@@ -0,0 +1,14 @@
+{
+  "homepage": "https://github.com/bazel-contrib/supply-chain",
+  "maintainers": [
+    {
+      "github": "Yannic"
+    },
+    {
+      "github": "TheGrizzlyDev"
+    }
```

```suggestion
    {
            "email": "antonio@engflow.com",
            "github": "TheGrizzlyDev",
            "name": "Antonio Di Stefano"
        },
        {
            "email": "yannic@engflow.com",
            "github": "Yannic",
            "name": "Yannic Bonenberger"
        }
```

We should also add a few others here (follow-up seems fine. putting it on the agenda for next week's meeting)